uint32 targetId

# Annotation type: -1 for unknown, 0 for partially visible, 1 for
# completely visible

int32 annotationType

sensor_msgs/RegionOfInterest bbox